rubicon_ml/viz/dataframe_plot.py

```diff
@@ -89,8 +89,15 @@ def load_experiment_data(self):
         `dataframe_name` must have the same schema.
         """
         self.data_df = None
+        count = 0
 
         for experiment in self.experiments:
+            if len(experiment.dataframes()) == 0:
+                print(
+                    f"WARNING: Experiment {experiment.name} does not have any dataframes logged to it. "
```

Comment:
experiments are generally referenced by ID instead of name as name is not required to be unique